core/execution/common/mark_price.go

```diff
@@ -222,9 +228,14 @@ func (mpc *CompositePriceCalculator) GetUpdateOraclePriceFunc(oracleIndex int) f
 			return err
 		}
 		p := mpc.scalingFunc(ctx, pd, mpc.oracles[oracleIndex].GetDecimals())
+		// ignore prices that exceed the cap
 		if p == nil || p.IsZero() {
 			return nil
 		}
+		// ignore price data > max price
+		if mpc.maxPrice != nil && p.GT(mpc.maxPrice) {
+			return nil
+		}
```

Comment:
Does this need to be ignored, or set to the max price?

Reply:
There's been some debate about this on protocol-design. The conclusion was to ignore values > max price

Reply:
right.